docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -130,6 +130,7 @@ The remainder of this section covers the tools we recommend for enabling this.
 You will need:
 
 * [Docker](https://docs.docker.com/engine/install/)
+  > you can use podman by changing the *CONTAINER_RUNTIME_ in the make file from docker to podman, [How to config podman socket](https://docs.podman.io/en/latest/markdown/podman-system-service.1.html#examples)
```

Comment:
where does it need to be?

Reply:
This section is about running _everything_ using Tilt. idk if Tilt even works with podman. 🤷 

There's a section much higher up, around line 12, about running tests and other contributor-facing tasks in a container.

Reply:
Here's how it can work with Tilt.

https://docs.tilt.dev/custom_build.html#buildah-or-any-image-builder-independent-of-docker

If you want to do that also, separate issue/PR please.